docs/guides/becoming-a-cna-as-an-open-source-org-or-project.md

```diff
@@ -0,0 +1,207 @@
+
+### Time and reporting obligations
+
+* **During CNA onboarding:**
+    * List of 3 available dates and times for a 1-hour call with CNA Program with all Points of Contact to answer questions and exercises. These three dates must be at least 3 weeks in the future.
+    * Complete practice exercises after the initial onboarding call and submit your exercises to the CNA Program for approval.
+
+* **Throughout CNA operations:**
+    * **No time requirements on timeline for [non-public aspects of coordinated vulnerability disclosure](https://www.cve.org/ResourcesSupport/AllResources/CNARules#appendix_d_disclosure_and_embargo_policies) (ie reporting, acknowledging, disclosures, etc).**
+    * CVE dispute requests have timeliness requirements:
+        * 3 days to acknowledge the dispute.
+        * 5 days after acknowledgement to make a decision or extend and inform the requester.
+        * 15 days after optional extension to make a final decision or escalate up to your Root, resulting in a new dispute cycle.
+    * Get approval from Root with changes to CNA record data like POCs, scope, vulnerability disclosure policy and location, etc.
+    * CNA inactivity policy: 6 months of no CVE IDs allocated will cause a few heartbeat emails sent to POCs. After three attempts to contact your organization will be removed from the CNA Program (and you can reapply to join again).
+    * Yearly review of CNA Rules and checking CNA mailing list (sent to POC email addresses).
+    * Recommended yearly rotation of CVE Services credentials.
+
+### Organization information
+
+* Full name of organization (“Python Software Foundation”)
+* Short name (ie “PSF”) used in CNA partner list and for authentication with CVE Services.
+* Industry using [Global Industry Classification Standard](https://en.wikipedia.org/wiki/Global_Industry_Classification_Standard) (GICS). Likely to be “451030” corresponding with “Software” for Open Source projects.
+* Country of Origin.
+* CNA Role & Type. Likely to be “Open Source” and “Vendor” for Open Source projects.
+
+### CNA and CVE information
+
+It’s acceptable to not have these fields complete at time of submission, they are finalized after the initial meeting and before announcing your organization as a CNA.
+
+* Root (recommended RedHat for Open Source projects)
+* Public contact information (ie email address, web form)
+* Scope statement
+* Vulnerability disclosure policy URL
+* Advisory location URL
+
+Optionally have the answers to this information:
+
+* Are there any in-flight CVE ID requests for your organization? (Let CVE know if there are, these can potentially be reassigned to your new CNA)
+* Approximate number of CVE IDs needed in a year (you can always request more)
+
+## Process to become a CNA
+
+### Contact RedHat to become a CNA under their Root
```

Comment:
I added this section as it seems RedHat is the recommended Root for Open Source projects. Would like some confirmation from CVE folks that this is indeed the case (we didn't get recommended this during the PSF's onboarding).

Reply:
Red Hat (with space).

I checked with others in the CVE Program recently, the short answer is that it's up to the incoming CNA. A new CNA can choose which Root to operate under, assuming agreement from the Root and a reasonable fit. So it's an option, probably a good one, for an OSS project to file with Red Hat.

Reply:
Done, thank you!